docs/supported_languages_and_lockfiles.md

```diff
@@ -18,9 +18,29 @@ nav_order: 2
 {:toc}
 </details>
 
-## Supported lockfiles
+### Core Concept
```

Comment:
can we call out to osv-scalibr here? as the underlying library for these?

Reply:
Made a call out to osv-scalibr here, though it is a bit janky. Perhaps we should have the osv-scanner and osv-scalibr integration better explained in another page.